templates/semantic-ui/zenodo_rdm/footer.html

```diff
@@ -27,58 +27,105 @@
+      </li>
+    </ul>
   </div>
   <div class="column">
-    <h4 class="ui inverted header">{{_('Blog')}}</h4>
-    <div class="ui inverted link list">
-      <a class="item" href="https://blog.zenodo.org">{{_('Blog')}}</a>
-    </div>
+    <h2 class="ui inverted small header">{{_('Blog')}}</h2>
+    <ul class="ui inverted link list">
+      <li class="item">
+        <a href="https://blog.zenodo.org">{{_('Blog')}}</a>
+      </li>
+    </ul>
   </div>
   <div class="column">
-    <h4 class="ui inverted header">{{_('Help')}}</h4>
-    <div class="ui inverted link list ">
-      <a class="item" href="https://help.zenodo.org">{{_('FAQ')}}</a>
-      <a class="item" href="https://help.zenodo.org/features">{{_('Features')}}</a>
-      <a class="item" href="https://help.zenodo.org/whatsnew">{{_("What's New")}}</a>
-    </div>
+    <h2 class="ui inverted small header">{{_('Help')}}</h2>
+    <ul class="ui inverted link list">
+      <li class="item">
+        <a href="https://help.zenodo.org">{{_('FAQ')}}</a>
+      </li>
+      <li class="item">
+        <a href="https://help.zenodo.org/features">{{_('Features')}}</a>
+      </li>
+      <li class="item">
+        <a href="https://help.zenodo.org/whatsnew">{{_("What's New")}}</a>
+      </li>
+      <li class="item">
+        <a href="https://about.zenodo.org/roadmap/">{{ _('Roadmap') }}</a>
+      </li>
+      <li class="item">
+        <a href="https://help.zenodo.org/guides/">{{ _('Guides') }}</a>
+      </li>
+      <li class="item">
+        <a href="https://help.zenodo.org/docs/">{{ _('Docs') }}</a>
+      </li>
+      <li class="item">
+        <a href="https://www.zenodo.org/support/">{{ _('Support') }}</a>
+      </li>
+    </ul>
   </div>
   <div class="column">
-    <h4 class="ui inverted header">{{_('Developers')}}</h4>
-    <div class="ui inverted link list ">
-      <a class="item" href="https://developers.zenodo.org">{{_('REST API')}}</a>
-      <a class="item" href="https://developers.zenodo.org#oai-pmh">{{_('OAI-PMH')}}</a>
-    </div>
+    <h2 class="ui inverted small header">{{_('Developers')}}</h2>
+    <ul class="ui inverted link list">
+      <li class="item">
+        <a href="https://developers.zenodo.org">{{_('REST API')}}</a>
+      </li>
+      <li class="item">
+        <a href="https://developers.zenodo.org#oai-pmh">{{_('OAI-PMH')}}</a>
+      </li>
+    </ul>
   </div>
   <div class="column">
-    <h4 class="ui inverted header">{{_('Contribute')}}</h4>
-    <div class="ui inverted link list ">
-      <a class="item" href="https://github.com/zenodo/zenodo"><i class="fa fa-external-link"></i> {{_('GitHub')}}</a>
-      <a class="item" href="/donate"><i class="fa fa-external-link"></i> {{_('Donate')}}</a>
-    </div>
+    <h2 class="ui inverted small header">{{_('Contribute')}}</h2>
+    <ul class="ui inverted link list">
+      <li class="item">
+        <a href="https://github.com/zenodo/zenodo"><i class="fa fa-external-link"></i> {{_('GitHub')}}</a>
```

Comment:
these are outdates icons from font awesome, maybe makes sense to change to semantic ui equivalents?

Reply:
We can make a different task for this?